src/thunderbots/software/ai/primitive/stop_primitive.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "ai/primitive/primitive.h"
+
+class StopPrimitive : public Primitive
+{
+    public:
+     static const std::string PRIMITIVE_NAME;
+
+    /**
+     * Creates a new Stop Primitive
+     *
+     * @param robot_id The id of the Robot to run this Primitive
+     * @param stop to stop the robot or not
+     */
+     explicit StopPrimitive(unsigned int robot_id, bool stop);
+
+    /**
+     * Creates a new Stop Primitive from a Primitive message
+     *
+     * @param primitive_msg The message from which to create the Stop Primitive
+     */
+     explicit StopPrimitive(const thunderbots_msgs::Primitive &primitive_msg);
+
+     std::string getPrimitiveName() const override;
+
+     unsigned int getRobotId() const override;
+
+     std::vector<double> getParameterArray() const override;
+
+     std::vector<bool> getExtraBitArray() const override;
+
+    private:
+     unsigned int robot_id;
+     bool stop;
+
+};
```

Comment:
Github complaining about lack of newline at end of file.

Reply:
bump?